<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="POS_TAB">
     <attribute name="title">
      <string>Position</string>
     </attribute>
     <widget class="QCustomPlot" name="POS_CHART" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>421</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="CAM_SELECTOR">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>231</width>
        <height>39</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Camera Set1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Camera Set2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>270</y>
        <width>16</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Z_LCD">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>250</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>180</y>
        <width>17</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Y_LCD">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="X_LCD">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>90</y>
        <width>16</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="IMU_TAB">
     <attribute name="title">
      <string>Raw IMU</string>
     </attribute>
     <widget class="QComboBox" name="IMU_SELECTOR">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Gyroscope</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Accelerometer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magnetometer</string>
       </property>
      </item>
     </widget>
     <widget class="QCustomPlot" name="IMU_CHART" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>581</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="IMU_X"/>
      </widget>
      <widget class="QSplitter" name="splitter_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="IMU_Y"/>
      </widget>
      <widget class="QSplitter" name="splitter_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="IMU_Z"/>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="FORCE_TAB">
     <attribute name="title">
      <string>Force</string>
     </attribute>
     <widget class="QcGaugeWidget" name="FORCE_GAUGE" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="FORCE_SELECTOR">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Force Sensor 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Force Sensor 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Force Sensor 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Force Sensor 4</string>
       </property>
      </item>
     </widget>
     <widget class="QLCDNumber" name="FORCE_LCD">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>140</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Force (N)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ENC_TAB">
     <attribute name="title">
      <string>Encoders</string>
     </attribute>
     <widget class="QLCDNumber" name="ENCODER1_LCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>2</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>72</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder 1</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="ENCODER2_LCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>182</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder 2</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="ENCODER3_LCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>222</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>292</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder 3</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="ENCODER4_LCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>332</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>402</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="L_RESET_BUTTON">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Length</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="L1_PRESET_VAL">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>100</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="L2_PRESET_VAL">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>180</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="L3_PRESET_VAL">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="L4_PRESET_VAL">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>340</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>140</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>L1 Preset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>220</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>L2 Preset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>300</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>L3 Preset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>628</x>
        <y>380</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>L4 Preset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="INIT_TAB">
     <attribute name="title">
      <string>Init Process</string>
     </attribute>
     <widget class="QQuickWidget" name="quickWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-11</y>
        <width>801</width>
        <height>451</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QcGaugeWidget</class>
   <extends>QWidget</extends>
   <header location="global">qcgaugewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
